tests/autopilot-capacity/aut_test.go

```diff
@@ -82,6 +83,10 @@ var autopilotruleScaleTestCases = []scheduler.AutopilotRuleParameters{
 	ruleResizeBy50IfPvcCapacityLessThan10,
 }
 
+var autopilotruleVolumeDriverDownTestCases = []scheduler.AutopilotRuleParameters{
```

Comment:
no need to create another list. Just use `autopilotruleBasicTestCases`. Any test case that gets added to `autopilotruleBasicTestCases` should also survive volume driver down.